consumer.go

```diff
@@ -91,10 +105,23 @@ func StartConsumer(config ConsumerConfig) (c *Consumer, err error) {
 		conns: make(map[string](chan<- Command)),
 	}
 
-	go c.run()
 	return
 }
 
+func StartConsumer(config ConsumerConfig) (c *Consumer, err error) {
+	c, err = NewConsumer(config)
+	if err != nil {
+		return
+	}
+
+	c.Start()
+	return
+}
+
+func (c *Consumer) Start() {
```

Comment:
The main reason I didn't design it this way initially is that I didn't want to figure out what the behavior should be if the consumer's `Start` method is called multiple times.

It seems that your implementation permits it, is that the expected behavior?